### PR TITLE
Add import to outputs

### PR DESCRIPTION
**Pull request by joraff**
Adds import resources to the table, tree, and json outputs. Uses Cyan color in tree view, and `(i)` to indicate an import.

### CODE REVIEW

**Review by dineshba — DISMISSED** [review]

**Comment by dineshba** [comment]
Thanks alot for the contribution @joraff

File: terraformstate/terraform_state.go [hunk 1]
```diff
@@ -19,9 +20,12 @@ type ResourceChange struct {
...
 	ProviderName  string `json:"provider_name"`
 	Change        struct {
-		Actions []string        `json:"actions"`
-		Before  json.RawMessage `json:"before,omitempty"`
-		After   json.RawMessage `json:"after,omitempty"`
+		Actions   []string        `json:"actions"`
+		Before    json.RawMessage `json:"before,omitempty"`
+		After     json.RawMessage `json:"after,omitempty"`
+		Importing struct {
+			Id string `json:"id"`
```
Comment (dineshba) [inline terraformstate/terraform_state.go:27]: @joraff golint is failing as it is Id to be `ID`. Can we change it ?

**Comment by joraff** [comment]
My pleasure! We get a lot of value from this tool, thank you for creating it.

File: terraformstate/terraform_state.go (hunk 1 above)
Comment (joraff) [reply terraformstate/terraform_state.go]: Of course, sorry about that

**Comment by joraff** [comment]
I didn't notice the linter before now, there are a few other issues to clean up. I'll get those fixed

**Review by dineshba — APPROVED** [review]

**Comment by joraff** [comment]
Resolves #47

**Pull request merged**